assets/src/scss/new-identity/style.scss

```diff
@@ -123,9 +123,12 @@ table.spreadsheet-table.is-color-gp-green {
 
 a:where(.standalone-link) {
   width: fit-content;
-  border-bottom: 2px solid var(--gp-green-500);
   font-weight: 700 !important;
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+}
+
+a:where(.standalone-link, .external-link, .pdf-link) {
```

Comment:
Let's try something like this to style links without classes
```
a:where(.standalone-link, .external-link, .pdf-link),
a:not([class]) {
...
}
```

Reply:
Before moving the styles to `theme.json` we used to define body link colors directly in [_typography.scss](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_typography.scss#L102-L108) using just the `a` selector, so we should be able to do that now with the new identity but that might require us to add a few changes to other links that have other styles